### ISSUE TITLE
Allow setting the animation curve for the list item sliding animation

### ISSUE THREAD

**Issue by Kwintenvdb** [wwwdata/implicitly_animated_reorderable_list#2]
Hi. Appreciate the work put into this package.

One thing I would love to see exposed in the public API is allowing the user to set the animation curve used for the sliding transition when elements are added / removed from the list. It seems that currently, elements move linearly, and only the duration of the animation can be controlled.

**Comment by wwwdata** [comment]
Yes, currently you could only adjust the curve with the wrapping animation widget that you use in the `itemBuilder`. Is there an issue with that? I mean you could just create your own curve like https://api.flutter.dev/flutter/animation/CurvedAnimation-class.html and use that in your animated widget of choice?

What would be the benefits if this is directly configured in the AnimatedList?
